catalog/plugin/catalog-plugin-federation-replication/src/main/java/ddf/catalog/federation/layered/replication/RestReplicatorPlugin.java

```diff
@@ -64,7 +65,7 @@ public CreateResponse process(CreateResponse input) throws PluginExecutionExcept
         Response r = client.post(data);
 
         LOGGER.debug("Posted the following GeoJSON: {}\n", data);
-        LOGGER.debug("RESPONSE: [{}]", ToStringBuilder.reflectionToString(r));
+        LOGGER.debug(RESPONSE, r);
```

Comment:
❓ Was the changed to use the regular toString() intentional? I suspect this was done because a Response object did not have a proper toString().